grpc/src/js_client/grpc_client.rs

```diff
@@ -0,0 +1,137 @@
+use wasm_bindgen::prelude::*;
+
+use celestia_types::blob::BlobParams;
+use celestia_types::block::Block;
+use celestia_types::state::auth::JsAuthParams;
+use celestia_types::state::auth::JsBaseAccount;
+use celestia_types::state::{JsCoin, TxResponse};
+
+use crate::grpc::GetTxResponse;
+use crate::grpc::TxStatusResponse;
+use crate::grpc::{GasInfo, JsBroadcastMode};
+use crate::Result;
+
+type InnerClient = crate::GrpcClient<tonic_web_wasm_client::Client>;
+
+/// Celestia GRPC client
+#[wasm_bindgen]
+pub struct GrpcClient {
+    client: InnerClient,
+}
+
+#[wasm_bindgen]
+impl GrpcClient {
+    /// Create a new client connected with the given `url`
+    pub async fn new(url: String) -> Result<Self> {
+        Ok(GrpcClient {
+            client: InnerClient::with_grpcweb_url(url),
+        })
+    }
+
+    /// Get auth params
+    pub async fn get_auth_params(&self) -> Result<JsAuthParams> {
+        Ok(self.client.get_auth_params().await?.into())
+    }
+
+    /// Get account
+    pub async fn get_account(&self, account: &str) -> Result<JsBaseAccount> {
+        Ok(self.client.get_account(&account.parse()?).await?.into())
+    }
+
+    /// Get accounts
+    pub async fn get_accounts(&self) -> Result<Vec<JsBaseAccount>> {
+        Ok(self
+            .client
+            .get_accounts()
+            .await?
+            .into_iter()
+            .map(Into::into)
+            .collect())
+    }
+
+    /// Get balance of coins with given denom
+    pub async fn get_balance(&self, address: &str, denom: String) -> Result<JsCoin> {
```

Comment:
I don't think that matters much but for consistency
```suggestion
    pub async fn get_balance(&self, address: &str, denom: &str) -> Result<JsCoin> {
```